packages/mobile/ios/Podfile.lock

```diff
@@ -491,22 +491,19 @@ DEPENDENCIES:
   - Yoga (from `../../../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
+  https://github.com/cocoapods/specs.git:
     - Analytics
     - boost-for-react-native
-    - Fabric
-    - Firebase
-    - FirebaseCore
-    - FirebaseInstanceID
-    - GoogleUtilities
```

Comment:
Looks like GoogleUtilities is now removed from this list? And Firebase too. The same thing happened on in my podfile.lock when I ran `bundle exec pod install`. Not sure if it will have any affect. @jeanregisser ?

Reply:
Looks like it's fine for now. I think it's still some leftover from when you ran it with a newer CocoaPods version.
We'll upgrade soon.